component/monitoring.jsonnet

```
@@ -31,6 +31,16 @@ local service_monitor = com.namespaced(params.namespace, {
   },
 });
 
+local asciiTitleCase(str) = std.asciiUpper(str[0]) + str[1:];
+
+local failed_job_alert_rules = std.map(
+  function(type) params.job_failed_alert_template {
+    alert: params.job_failed_alert_template.alert % { type: asciiTitleCase(type) },
+    expr: params.job_failed_alert_template.expr % { type: type },
```

These might work as just

```suggestion
    alert: super.alert % { type: asciiTitleCase(type) },
    expr: super.expr % { type: type },
```

but I'm never sure in which contexts Jsonnet provides `super`.

---

works indeed! TIL